qemu/tests/cfg/openflow_acl_test.cfg

```diff
@@ -15,11 +15,14 @@
             prepare_cmd_linux = echo "autotest page" > /var/www/html/autotest-test.html
             cleanup_cmd_linux = rm -rf /var/www/html/autotest-test.html
             access_cmd_linux = curl -L http://ACCESS_TARGET/autotest-test.html --connect-timeout 10
-            access_cmd_windows = cd C:\lynx && C:\lynx\lynx.exe --dump http://ACCESS_TARGET/autotest-test.html
+            curl_win_path = C:\curl\
+            access_cmd_windows = cd C:\curl && curl.exe -L http://ACCESS_TARGET/autotest-test.html
             setup_cmd_windows = start /w pkgmgr /iu:IIS-WebServerRole;IIS-WebServer
             prepare_cmd_windows = echo "autotest page" > C:\inetpub\wwwroot\autotest-test.html
             cleanup_cmd_windows =
             stop_cmd_windows =
+            Windows:
+                copy_curl = curl.exe ca-bundle.crt
```

Comment:
Could you please explain why it needs `ca-bundle.crt`? I would suggest using option `-k` to skip cert check if it is unnecessary.